src/Http/Http/src/Features/RequestServicesFeature.cs

```diff
@@ -2,12 +2,9 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Diagnostics;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Http.Features;
 using Microsoft.Extensions.DependencyInjection;
 
-namespace Microsoft.AspNetCore.Hosting.Internal
+namespace Microsoft.AspNetCore.Http.Features
 {
     public class RequestServicesFeature : IServiceProvidersFeature, IDisposable
```

Comment:
internal?

Reply:
The other features are all public so, putting this one as public too.